tests/review.t

```diff
@@ -34,11 +34,15 @@ We can also install these updates automatically:
   $ pip-review
   Everything up-to-date
 
-Next, let's test for regressions with older versions of pip:
+Next, let's test for regressions with older versions of pip and Python:
 
   $ pip install --force-reinstall --upgrade pip\<6.0 >/dev/null 2>&1
-  $ pip-review
-  Everything up-to-date
+  $ if python -c 'import sys; sys.exit(0 if sys.version_info <= (3,5) else 1)'; then
+  >   pip-review
+  > else
+  >   echo Skipped
+  > fi
+  (Everything up-to-date|Skipped) (re)
 
```

Comment:
This is a useful bit. pip<6.0 won't install with the newest versions of Python, so skipping this test is justified in that case. It doesn't hurt to have this in the tests even when we aren't testing Python 3 yet, so I would happily accept this bit in a separate pull request.